### ISSUE TITLE
Twitter files are empty

### ISSUE THREAD

**Issue by yhjhoo** [trishagee/sense#4]
As the title, is it intend to be empty?

**Comment by trishagee** [comment]
Yes, they are - they're too big to upload to GitHub, unfortunately.  The [tweetdata-for-testing.txt](https://github.com/trishagee/sense/blob/master/sense/src/test/resources/tweetdata-for-testing.txt) file is the best thing you can use for trying out the app.
